packages/gestalt-datepicker/src/DatePicker.css

```diff
@@ -359,22 +359,41 @@ html:not([dir="rtl"]) :global ._gestalt .react-datepicker__navigation--next {
   z-index: 1;
 }
 
-.react-datepicker-popper[data-placement^="bottom"] {
-  margin-top: 6px;
-}
+/* Year / Month dropdown  */
 
-.react-datepicker-popper[data-placement^="top"] {
-  /* !important is required to keep same distance from Textfield
-     when popper flips from top to bottom */
-  top: -2px !important; /* stylelint-disable-line declaration-no-important */
+:global ._gestalt .react-datepicker__month-select,
+:global ._gestalt .react-datepicker__year-select {
+  background-color: var(--g-colorGray0);
+  border-color: var(--color-border-container);
+  border-radius: 16px;
+  border-style: solid;
+  border-width: 2px;
+  color: var(--color-text-subtle);
+  cursor: pointer;
+  font-family: var(--font-family-default-latin);
+  font-size: var(--font-size-200);
+  height: 40px;
+  margin-bottom: 10px;
+  margin-top: 10px;
+  padding: 5px 16px; /* right padding prevents it from running into arrow */
```

Comment:
This is applying padding to both left and right sides…is that intentional? If so, you might want to update the comment to something like "horizontal padding prevents…"

Also, does this work fine with RTL?